### DISCUSSION TITLE
v6 tag pointing at 6.11.4 instead of 6.11.5 at https://www.npmjs.com/package/react-table

### DISCUSSION THREAD

**Discussion by dje1990** [TanStack/table#2104] [General]
At the npm site
https://www.npmjs.com/package/react-table

in the Versions tab, 
The v6 tag is pointing at 6.11.4 instead of 6.11.5
Is this a mistake?

**Comment by cristianocca** [comment]
Looks like v6 also points to an outdated v6 branch:

"react-table-v6": "6.8.6" is the current latest on npm, but 6.11.5 is supposed to be the latest version?  @tannerlinsley can you take a quick a look at it? Some people are stuck with v6 :(

**Comment by tannerlinsley** [comment]
- Are you using v6 and v7 in parallel? If not, then just use `react-table@6`, which should be fine.

**Comment by dje1990** [comment]
Hi @tannerlinsley in my package.json I am using `"react-table": "6.11.5"` and that is working fine, but when I was checking for versions on NPM I noticed that the website was incorrect and raised this ticket to fix that. Thanks.

**Comment by cristianocca** [comment]
@tannerlinsley  it is still good to use `react-table-v6` as the dependency so we don't get it labelled as outdated when running npm-outdated.  Can you update the npm's repository so it points to the latest version of the 6 series?

**Comment by Hopp3r** [comment]
I've found two npm libs associated to v6. The one that v6 docs point to is 6.8.6 while the other is 6.11.0.
What is the appropriate & latest version of v6 lib to use that is still somewhat community maintained while I migrate my project to v7?
I would like to run v6 and v7 in parallel while I'm handling migration.

V6 docs point to `react-table-v6` package, but there's a second one that appears to be a later version. This is confusing.

https://www.npmjs.com/package/react-table-v6  `version 6.8.6`
https://www.npmjs.com/package/react-table-6    `version 6.11.0`
